financial-services-accelerator/accelerators/fs-is/repository/conf/configure.properties

```diff
@@ -33,9 +31,8 @@ DB_PASS=root
 DB_HOST=localhost
 DB_DRIVER=com.mysql.jdbc.Driver
 
-DB_APIMGT=fs_apimgtdb
+DB_IDENTITY=fs_identitydb
```

Comment:
This is a shared DB between two products right? Shouldn't the DB name be the same in two accelerators?